flixel/tweens/FlxEase.hx

```diff
@@ -151,25 +155,60 @@ class FlxEase
 
 	public static function bounceIn(t:Float):Float
 	{
-		return 1 - bounceOut(1 - t);
+		return BOUNCE_IN(t);
+	}
+
+	public static function bounceInCustom(/*bounces:Int = 6, */?Bounce1:Float, ?Bounce2:Float, ?Bounce3:Float, ?Bounce4:Float, ?Bounce5:Float, ?Bounce6:Float):EaseFunction
+	{
+		var outFunc:EaseFunction = bounceOutCustom(/*bounces, */Bounce1, Bounce2, Bounce3, Bounce4, Bounce5, Bounce6);
+		var func:EaseFunction = function(t:Float)
+		{
+			return 1 - outFunc(1 - t);
+		};
+		return func;
+	}
+
+	public static function bounceOutCustom(/*bounces:Int = 6, */?Bounce1:Float, ?Bounce2:Float, ?Bounce3:Float, ?Bounce4:Float, ?Bounce5:Float, ?Bounce6:Float):EaseFunction
+	{
+		Bounce1 = Bounce1 == null ? DEFAULT_B1 : Bounce1;
+		Bounce2 = Bounce2 == null ? DEFAULT_B2 : Bounce2;
+		Bounce3 = Bounce3 == null ? DEFAULT_B3 : Bounce3;
+		Bounce4 = Bounce4 == null ? DEFAULT_B4 : Bounce4;
+		Bounce5 = Bounce5 == null ? DEFAULT_B6 : Bounce5;
+		Bounce6 = Bounce6 == null ? DEFAULT_B6 : Bounce6;
+		var func:EaseFunction = function(t:Float)
+		{
+			if (t < Bounce1)
+				return 7.5625 * t * t;
+			if (t < Bounce2)
+				return 7.5625 * (t - Bounce3) * (t - Bounce3) + .75;
+			if (t < Bounce4)
+				return 7.5625 * (t - Bounce5) * (t - Bounce5) + .9375;
+			return 7.5625 * (t - Bounce6) * (t - Bounce6) + .984375;
+		};
+		return func;
+	}
```

Comment:
1. remove commented args
2. use lowercase args
3. use default arg values  from consts instead of null floats, this will work if B1-B6 are changed to inline
4. define an inlined private helper rather than using anonymous functions
```suggestion
		return 1 - bounceOutHelper(t);
	}
	
	public static function bounceInCustom(b1 = B1, b2 = B2, b3 = B3, b4 = B4, b5 = B5, b6 = B6):Float
	{
		return (t)->1 - bounceOutHelper(t, b1, b2, b3, b4, b5, b6);
	}
	
	public static function bounceOutCustom(b1 = B1, b2 = B2, b3 = B3, b4 = B4, b5 = B5, b6 = B6):Float
	{
		return (t)->bounceOutHelper(t, b1, b2, b3, b4, b5, b6);
	}
	
	static inline function bounceOutHelper(t:Float, b1 = B1, b2 = B2, b3 = B3, b4 = B4, b5 = B5, b6 = B6):Float
	{
			if (t < b1)
				return 7.5625 * t * t;
			if (t < b2)
				return 7.5625 * (t - b3) * (t - b3) + .75;
			if (t < b4)
				return 7.5625 * (t - b5) * (t - b5) + .9375;
			return 7.5625 * (t - b6) * (t - b6) + .984375;
	}
```